pkg/jobs/registry.go

```diff
@@ -1309,15 +1309,12 @@ func (r *Registry) cancelRequested(ctx context.Context, txn isql.Txn, id jobspb.
 func (r *Registry) PauseRequested(
 	ctx context.Context, txn isql.Txn, id jobspb.JobID, reason string,
 ) error {
-	job, resumer, err := r.getJobFn(ctx, txn, id)
+	job, _, err := r.getJobFn(ctx, txn, id)
```

Comment:
nit: you could use `LoadJobWIthTxn`

Reply:
:D Yeah, have a commit locally that removes getJobFn completely. I'll push that after merging this.